ls1intum/Artemis
# Remove back arrow from course management

### Checklist

### Motivation and Context



Breadcrumb navigation is used in all `Course Management` and `Server Administration` pages. This replaces the "back arrow" so we remove those from all pages.

### Description


Remove the back arrows from all sub-pages.

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Select Assessment Board
4. Back arrow should be gone in that view
4. Click on Complaints about your assesement
5. Back arrow should be gone in that view
6. Go back and click on Assesements locked by you
7. Back arrow should be gone in that view


### Test Coverage



Doesn't change since only HTML templates are affected. 
### Screenshots


![Screenshot from 2020-11-22 11-52-43](https://user-images.githubusercontent.com/11298674/99901991-d1c7bf00-2cba-11eb-846c-d69df7a9263c.png)
![Screenshot from 2020-11-22 11-53-02](https://user-images.githubusercontent.com/11298674/99901994-d3918280-2cba-11eb-9716-84294f8bf4ce.png)
![Screenshot from 2020-11-22 11-53-34](https://user-images.githubusercontent.com/11298674/99901996-d55b4600-2cba-11eb-8ee4-14177ea39641.png)
